app/controllers/specialist_documents_controller.rb

```diff
@@ -15,39 +15,25 @@ def show
 
 private
 
-  def finder(slug)
-    Finder.new(content_store.content_item("/#{slug}"))
+  def document(finder, artefact)
+    case finder_slug
+    when "drug-safety-update"
```

Comment:
Can we use the document format here instead? It feels more robust to special case `drug_safety_update` documents than to special case the finder they belong to.